File: app_java/src/main/java/ru/rbs/mobile/payment/sample/java/MarketApplication.java
package ru.rbs.mobile.payment.sample.java;

import android.app.Application;
import android.content.Context;

import ru.rbs.mobile.payment.sdk.SDKPayment;
import ru.rbs.mobile.payment.sdk.component.impl.CachedKeyProvider;
import ru.rbs.mobile.payment.sdk.component.impl.RemoteKeyProvider;

public class MarketApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        SDKPayment.INSTANCE.init(this);
    }
}

File: app_kotlin/src/main/kotlin/ru/rbs/mobile/payment/sample/kotlin/MarketApplication.kt
package ru.rbs.mobile.payment.sample.kotlin

import android.app.Application
import ru.rbs.mobile.payment.sdk.SDKConfigBuilder
import ru.rbs.mobile.payment.sdk.SDKPayment
import ru.rbs.mobile.payment.sdk.component.impl.RemoteCardInfoProvider
import ru.rbs.mobile.payment.sdk.component.impl.RemoteKeyProvider

class MarketApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        SDKPayment.init(this)
    }
}